src/definitions/behaviors/form.js

```diff
@@ -545,18 +545,18 @@ $.fn.form = function(parameters) {
               name
             ;
             if(requiresValue) {
-              prompt = prompt.replace('{value}', $field.val());
+              prompt = prompt.replace(/\{value\}/g, $field.val());
             }
             if(requiresName) {
               $label = $field.closest(selector.group).find('label').eq(0);
               name = ($label.length == 1)
                 ? $label.text()
                 : $field.prop('placeholder') || settings.text.unspecifiedField
               ;
-              prompt = prompt.replace('{name}', name);
+              prompt = prompt.replace(/\{name\}/g', name);
```

Comment:
Remove the single quote, which breaks the script. It should read:
```javascript
prompt = prompt.replace(/\{name\}/g, name);
```

Reply:
Fixed. Apologies. This is my first contribution. Silly mistake. Nerves, I think.

Reply:
Just test your code before submitted a PR 😉 don't do "dry" PRs, because you think it will work

Reply:
Understood. Agreed. That's one of the underlying reasons I was apologizing